go-controller/pkg/types/const.go

```diff
@@ -255,11 +255,11 @@ const (
 	// InformerSyncTimeout is used when waiting for the initial informer cache sync
 	// (i.e. all existing objects should be listed by the informer).
 	// It allows ~4 list() retries with the default reflector exponential backoff config
-	InformerSyncTimeout = 20 * time.Second
+	InformerSyncTimeout = 60 * time.Second
 
 	// HandlerSyncTimeout is used when waiting for initial object handler sync.
 	// (i.e. all the ADD events should be processed for the existing objects by the event handler)
-	HandlerSyncTimeout = 20 * time.Second
+	HandlerSyncTimeout = 60 * time.Second
```

Comment:
this as the comment suggests is the time it takes for "adding the first set of objects we get ?" so to finish setup for 4000 efw?

Reply:
Only in my test environment. It is hard to give an exact number or a formula to calculate the time needed.